microverse-repo2/project2.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Second Repo</title>
+    <style>
+    div.main-body {
+	display: grid;
+	grid-template-columns: minmax(300px , auto)1fr;
+	grid-template-areas: "navbar mainContent";
+	grid-gap: 20px;
+    }
+
+nav#navbar {
+	grid-area: navbar;
+	position: fixed;
+}
+
+nav#navbar a {
+	display: block;
+	border: 1px solid black;
+	padding: 5px;
+	margin: 10px 0;
+	text-decoration: none;
+	color: black;
+}
+
+main#main-doc {
+	grid-area: mainContent;
+}
+
+header {
+	font-size: 1.5rem;
+	font-weight: bold;
+}
+
+code {
+	background-color: #CCC;
+	display: block;
+	padding: 20px;
+}
+@media screen and (min-width: 480px) {
+  body {
+    background-color: lightgreen;
+  }
+}
+    </style>
+</head>
+
+<body>
+    <div class="main-body">
```

Comment:
Kindly add a section here.

Reply:
@norbiboy1 **great job, now kindly try always to include an explanation, a "why", for each requested change. For example:**

> Why don't you use a section tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible